### PR TITLE
Completion of Project

### PR DESCRIPTION
**Pull request by phfdonda**
I believe the project is complete. I hope you can point out any ways in which I can improve. Thank you!

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
## Hi @phfdonda !  
### Great job!

#### General


* [ ] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
* You .stickler.yml file is empty
Screenshot from 2020-03-24 00-03-30

* Please fix this
Screenshot from 2020-03-24 00-02-38



#### Section specific

* [ ] Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide  from Pathwright
screenshot 



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

**Comment by phfdonda** [comment]
Hi, Anna, thank you for spotting that! I was having problems copying the file, so I had decided to copy the content and just pasting and saving, but apparently saved only part of it. Now it's fixed!

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+def bubble_sort(array)
+
+  size = array.length - 1
+  loop do
+    swapped = false
+    0.upto(size).each do |n|
```
Comment (kobiyoyo) [inline bubble_sort.rb]: its more efficient written like this `(size).times do `

**Review by kobiyoyo — APPROVED** [review]
# Project Approved :thumbsup:
## Hello @phfdonda  ,  You have done a great job with this project. :clap: but there is one more thing to do .
check comments above

File: bubble_sort.rb (hunk 1 above)
Comment (phfdonda) [reply bubble_sort.rb]: Oh, yeah, since it starts from 0, right? Haha, how dumb of me! Thank you for pointing that!

**Pull request merged**